### DISCUSSION TITLE
Linux: SSL Certificate verify failed

### DISCUSSION THREAD

**Discussion by Nagasairam112** [ansys/pydpf-core#1444] [General]
I am helping a customer who is accessing the pydpf core examples on a Linux cluster with correctly configured venv , when he is accessing the examples from default examples he is getting the SSL certification issues as in the following image. 
![HDJ0457u9k](https://github.com/ansys/pydpf-core/assets/61224632/8a9a2063-e3a4-4209-8a9a-5404727d0657)

any comments or thoughts on why this is happing and how to fix it? 
Cheers, Ram

**Comment by ansys-akarcher** [comment]
Hello @Nagasairam112 ,

In pydpf-core some examples files are downloaded from a public github repo,
what seems to be failing is the Certificate verification of the HTTPS request.

From official pip documentation: https://pip.pypa.io/en/stable/topics/https-certificates/
Python seems to be using predefined certificates from the package certifi
**So the first suggestion would be to try**: `pip install certifi --upgrade` (or pip3)

If this doesn't work, then the user may want to use some custom specific certificates or the one from his system.
This information is available on the link I provided.

Please tag me again if there is still an issue.

**Comment by Nagasairam112** [comment]
Thank you Antoine,

I will update you and keep you posted.




Naga sai ram .Gopisetti
________________________________
